src/frontend/src/utils/types.ts

```diff
@@ -22,6 +23,12 @@ export interface LinkItem {
   route: string;
 }
 
+export interface MUILinkItem {
+  name: string;
+  icon?: typeof SvgIcon;
+  route: string;
+}
+
```

Comment:
If LinkItem interface is not used anywhere else in the codebase might be a good idea to take it out. Won't be needed anymore since it uses font awesome icons